R/data.R

```diff
@@ -0,0 +1,78 @@
+#' Synthetic dataset of stochastic SIR system with known Rt
+#'
+#' A dataset produced by Katelyn Gostic as part of a published manuscript:
+#' Gostic, Katelyn M., et al. "Practical considerations for measuring the
+#' effective reproductive number, R t." PLoS computational biology 16.12 (2020):
+#' e1008409. The data are simulated from a stochastic SEIR compartment model.
+#'
+#' This synthetic dataset has a number of desirable properties: 1. The force of
+#' infection is manipulated by changing the Rt, allowing for sudden changes in
+#' infection dynamics. This behavior is a strong test of the flexibility of an
+#' Rt estimation framework 2. The realized Rt is known at each timepoint 3. The
+#' dataset incorporates a simple generation interval and a reporting delay
+#'
+#' In their manuscript, Gostic et al. benchmark the performance of a number of
```

Comment:
```suggestion
#' Gostic et al. benchmark the performance of a number of
```

No need to say "in their manuscript"